Classes/Mvc/Property/TypeConverter/UploadedFileReferenceConverter.php

```diff
@@ -204,7 +200,14 @@ protected function importUploadedResource(array $uploadInfo, PropertyMappingConf
             }
         }
 
-        $uploadFolderId = $configuration->getConfigurationValue(UploadedFileReferenceConverter::class, self::CONFIGURATION_UPLOAD_FOLDER) ?: $this->defaultUploadFolder;
+        /** @var \TYPO3\CMS\Extbase\Object\ObjectManager $objectManager */
+        $objectManager = \TYPO3\CMS\Core\Utility\GeneralUtility::makeInstance('TYPO3\CMS\Extbase\Object\ObjectManager');
+        /** @var \TYPO3\CMS\Extensionmanager\Utility\ConfigurationUtility $configurationUtility */
+        $configurationUtility = $objectManager->get('TYPO3\CMS\Extensionmanager\Utility\ConfigurationUtility');
+        $moduleConfiguration = $configurationUtility->getCurrentConfiguration('userimport');
+        $uploadStorageFolder = $moduleConfiguration['uploadStorageFolder']['value'] != '' ? $moduleConfiguration['uploadStorageFolder']['value'] : $moduleConfiguration['uploadStorageFolder']['default_value'];
```

Comment:
Use `empty($moduleConfiguration['uploadStorageFolder']['value'])` instead of `$moduleConfiguration['uploadStorageFolder']['value'] != ''` or `$moduleConfiguration['uploadStorageFolder']['value'] !== ''` (strict comparison).